ister.py

```diff
@@ -503,14 +504,20 @@ def create_mount_unit(filename, uuid, mount, fs_type):
             cmd = "mount {0}{1} {2}{3}".format(dev, pnum, target_dir,
                                                part["mount"])
             run_command(cmd)
-        if part["mount"] not in ["/", "/boot", "/srv", "/home"]:
-            if not part["mount"].startswith("/usr"):
-                filename = part["mount"][1:].replace("/", "-") + ".mount"
-                if not os.path.exists(units_dir):
-                    os.makedirs(units_dir)
-                create_mount_unit(os.path.join(units_dir, filename),
-                                  get_uuid(pnum, base_dev),
-                                  part["mount"], fs_type)
+
+        # Create mount units for the partitions, except for those having standard
+        # GPT type GUIDs, because the standard systemd 'systemd-gpt-auto-generator'
+        # tool will generate the mount points.
+        if part["mount"] in ["/", "/boot", "/srv", "/home"]:
+            continue
+        if part["mount"].startswith("/usr"):
```

Comment:
Not your fault but would you mind fixing this case so that /usr-2 isn't skipped as we really only want to verify not mounting over /usr/ parts by default.